src/wp-includes/class-wp-theme.php

```diff
@@ -464,11 +483,13 @@ public function __construct( $theme_dir, $theme_root, $_child = null ) {
 
 		// We're good. If we didn't retrieve from cache, set it.
 		if ( ! is_array( $cache ) ) {
+			$this->is_block_theme();
```

Comment:
@mukeshpanchal27 Why does this need to be changed? I think it's correct as is here, since this may be a valid theme, where we need to check whether it's a block theme.

Reply:
Thanks @felixarntz for pointing it out. I miss it. going to close this comment.